packages/browser-wallet/src/background/index.ts

```diff
@@ -111,6 +111,10 @@ chrome.storage.local.onChanged.addListener((changes) => {
 
 chrome.runtime.onStartup.addListener(startupHandler);
 chrome.runtime.onInstalled.addListener(startupHandler);
+bgMessageHandler.handleMessage(createMessageTypeFilter(InternalMessageType.PopupReady), (_i, _s, respond) => {
```

Comment:
So before the popup sent out the PopupReady to no one, and didn't expect an answer? (Why are we now telling it to wait for a response then?)

Reply:
PopupReady was being used when spawning a new window to ensure the window is actually available and ready, before we try to trigger stuff in the popup from the background script (`background/window-management.ts` line 39).

But I guess you have a point that I shouldn't send it an answer, but merely start the confirmation process.